src/ocean/util/encode/Base64.d

```diff
@@ -553,6 +553,19 @@ unittest
         test!("==")(result, decode("TWE="));
         test!("!=")(result, decode(""));
     }
+
+    // don't decode short things at the end
+    {
+        auto result = cast(Const!(ubyte)[])"Maq";
+        // B should be ignored since it's missing the second byte
+        test!("==")(result, decode("TWFxB"));
```

Comment:
> What is a *short thing*

https://github.com/sociomantic-tsunami/ocean/pull/193#issue-243352284 contains the answer. Suggest adding it in here as a code comment.